### PR TITLE
Added HKActivitySummary extensions

### PR DESCRIPTION
**Pull request by lhygilbert**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 5.0.
- [x] New extensions support iOS 10.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+, or use `@available` if not.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+// MARK: - Properties
+@available(watchOS 2.2, *)
+public extension HKActivitySummary {
+
+    /// SwifterSwift: Check if stand goal is met
+    var isStandGoalMet: Bool {
+        let unit = HKUnit.count()
+        let standHours = appleStandHours.doubleValue(for: unit)
+        let goal = appleStandHoursGoal.doubleValue(for: unit)
+        return standHours >= goal
```
Comment (LucianoPAlmeida) [inline Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift:21]: I don't know this API ... but there is the possibility of not having a goal set? what happens in this situation in `doubleValue` return?

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import HealthKit
+
+// MARK: - Properties
+@available(watchOS 2.2, *)
+public extension HKActivitySummary {
+
+    /// SwifterSwift: Check if stand goal is met
+    var isStandGoalMet: Bool {
+        let unit = HKUnit.count()
+        let standHours = appleStandHours.doubleValue(for: unit)
```
Comment (LucianoPAlmeida) [inline Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift:19]: Is there a way to compare this without using an arbitrary unit? 
I don't know the API, so these are only questions :)

Comment (lhygilbert) [reply Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift:19]: Unfortunately no =[

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift (hunk 1 above)
Comment (lhygilbert) [reply Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift:21]: In real life, there should be always a goal when fetching `HKActivitySummary` from HealthKit

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift (hunk 2 above)
Comment (lhygilbert) [reply Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift:19]: Sorry, I just did doing more research on this, turns out there is a way to compare this without using an arbitrary unit. Just push the update

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import HealthKit
+
+// MARK: - Properties
+@available(watchOS 2.2, *)
+public extension HKActivitySummary {
+
+    /// SwifterSwift: Check if stand goal is met
+    var isStandGoalMet: Bool {
+        return appleStandHoursGoal.compare(appleStandHours) != .orderedDescending
+    }
```
Comment (guykogus) [inline Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift:19]: You can make these all 1-liners.
```suggestion
    var isStandGoalMet: Bool { appleStandHoursGoal.compare(appleStandHours) != .orderedDescending }
```

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+    /// SwifterSwift: Check if exercise time goal is met
+    var isExerciseGoalMet: Bool {
```
Comment (guykogus) [inline Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift:22]: I think the names should match more exactly.
```suggestion
    var isExerciseTimeGoalMet: Bool {
```

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        return appleExerciseTimeGoal.compare(appleExerciseTime) != .orderedDescending
+    }
+
+    /// SwifterSwift: Check if active energy goal is met
+    var isEnergyGoalMet: Bool {
```
Comment (guykogus) [inline Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift:27]: Same here
```suggestion
    var isEnergyBurnedGoalMet: Bool {
```

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift (hunk 3 above)
Comment (lhygilbert) [reply Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift]: Thanks for the suggestion, updated

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift (hunk 4 above)
Comment (lhygilbert) [reply Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift]: updated

File: Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift (hunk 5 above)
Comment (lhygilbert) [reply Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift]: updated

File: CHANGELOG.md [hunk 6]
```diff
@@ -22,6 +22,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - Added new `+`, `+=`, `-` and `-=` operator extensions for tuple (width: CGFloat, height: CGFloat). [#841](https://github.com/SwifterSwift/SwifterSwift/pull/841) by [hamtiko](https://github.com/hamtiko)
 - **WKWebView**:
   - Added `loadURL(_:)` and `loadURLString(_:)` extensions for `WkWebView`. [#851](https://github.com/SwifterSwift/SwifterSwift/pull/851) by [hamtiko](https://github.com/hamtiko)
+- **HKActivitySummary**:
+  - Added `isStandGoalMet`, `isExerciseGoalMet`, and `isEnergyGoalMet`. [#875](https://github.com/SwifterSwift/SwifterSwift/pull/875) by [lhygilbert](https://github.com/lhygilbert)
```
Comment (guykogus) [inline CHANGELOG.md:26]:
```suggestion
  - Added `isStandGoalMet`, `isExerciseTimeGoalMet`, and `isEnergyBurnedGoalMet `. [#875](https://github.com/SwifterSwift/SwifterSwift/pull/875) by [lhygilbert](https://github.com/lhygilbert)
```

**Comment by SD10** [comment]
Thank you for contributing to SwifterSwift! I've invited you to join the SwifterSwift GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md). Feel free to reach out if you have any questions! 😃

**Pull request merged**